go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/hashicorp/golang-lru
 
-go 1.12
+go 1.19
```

Comment:
Two things here:

1) The module name needs to be updated to `github.com/hashicorp/golang-lru/v2`
2) Is Go 1.19 specifically needed or does Go 1.18 suffice? I think it's fine in `/v2` to require 1.19 if it's really needed but wanted to check.

Reply:
I changed it to 1.18